.gitlab-ci.yml

```diff
@@ -123,72 +142,62 @@ test-linux-stable:                 &test
     - ./scripts/gitlab/test_linux_stable.sh
     - sccache -s
 
-check-web-wasm:                    &test
+check-web-wasm:
   stage:                           test
-  <<:                              *test-refs
+  <<:                              *rules-test
   <<:                              *docker-env
-  <<:                              *compiler_info
+  <<:                              *compiler-info
   script:
-    # WASM support is in progress. As more and more crates support WASM, we
-    # should add entries here. See https://github.com/paritytech/polkadot/issues/625
+    # WASM support is in progress. As more and more crates support WASM, we should
+    # add entries here. See https://github.com/paritytech/polkadot/issues/625
     - ./scripts/gitlab/check_web_wasm.sh
     - sccache -s
 
-check-runtime-benchmarks:          &test
+check-runtime-benchmarks:
   stage:                           test
-  <<:                              *test-refs
+  <<:                              *rules-test
   <<:                              *docker-env
-  <<:                              *compiler_info
+  <<:                              *compiler-info
   script:
     # Check that the node will compile with `runtime-benchmarks` feature flag.
     - ./scripts/gitlab/check_runtime_benchmarks.sh
     - sccache -s
 
+#### stage:                        build
+
 check-transaction-versions:
```

Comment:
doesn't this need a `docker-env` or `kubernetes-env`?

Reply:
thanks for noticing!